package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "probot",
-  "version": "0.0.0-development",
+  "version": "9.8.1",
```

Comment:
the version is set automatically by [semantic-release](https://github.com/semantic-release/semantic-release) right before publishing to npm. We do not commit the version to git. Can you undo this change, please?

When adding update-notifier, you can maybe ignore it if the version is `0.0.0-development` as it means someone is using a local version instead of one from npm